app/src/main/java/org/wikipedia/page/PageProperties.java

```diff
@@ -190,7 +191,7 @@ public void writeToParcel(Parcel parcel, int flags) {
         parcel.writeLong(revisionId);
         parcel.writeLong(lastModified.getTime());
         parcel.writeString(displayTitleText);
-        parcel.writeString(GeoMarshaller.marshal(geo));
+        parcel.writeParcelable(geo, Parcelable.PARCELABLE_WRITE_RETURN_VALUE);
```

Comment:
Feels like we do not really need to `marshal(geo)` here, and can be replaced with `writeParcelable(geo, Parcelable.PARCELABLE_WRITE_RETURN_VALUE)`.

Reply:
What about when `geo == null`?  (which is why the tests fail)

Reply:
Yes, you are correct, and thanks for committing the changes.